main/src/ca/uwaterloo/flix/util/ParOps.scala

```diff
@@ -17,30 +17,38 @@
 package ca.uwaterloo.flix.util
 
 import java.util
-import java.util.concurrent.{Callable, Executors}
-
+import java.util.concurrent.{Callable, Future}
 import scala.jdk.CollectionConverters._
-import scala.collection.parallel._
 import ca.uwaterloo.flix.api.Flix
 
+import scala.collection.mutable.ArrayBuffer
+
 object ParOps {
 
   /**
     * Apply the given function `f` to each element in the list `xs` in parallel.
     */
   @inline
   def parMap[A, B](xs: Iterable[A])(f: A => B)(implicit flix: Flix): Iterable[B] = {
-    // Build the parallel array.
-    val parArray = xs.toParArray
-
-    // Configure the task support.
-    parArray.tasksupport = flix.forkJoinTaskSupport
+    val out = ArrayBuffer.fill(xs.size)(null.asInstanceOf[B])
```

Comment:
Any reason to use an `ArrayBuffer` over a raw array? 

The array will never change its size anyway.

Reply:
It's possible in `parAgg`, but in this particular case it requires a `B: ClassTag` bound which seems unnecessarily noisy.